images/linux/toolsets/toolset-2004.json

```diff
@@ -231,6 +231,8 @@
             "alpine:3.12",
             "alpine:3.13",
```

Comment:
Are we going to remove old versions? Please see the [announcement](https://github.com/actions/runner-images/issues/6317)

Reply:
This is only the first part of the changes that are not breaking. It's too early to delete old images.